# sensor_range and sensor_fov should really not be part of this message

Header header
float32 sensor_range
float32 sensor_fov
geometry_msgs/Pose[] people